scripts/ci/sign-new-macos-releases.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+set -e
+
+echo "::group::Unpack any new darwin arm64 and amd64 binaries to ./tmp"
+    # ./releases/{DIST_NAME}/{DIST_VERSION}/*_darwin-${arch}.tar.gz
+    # -> ./tmp/${DIST_NAME}_${DIST_VERSION}_${arch}-unsigned/
+    for NEW_DIR in ./releases/*/*; do
+        (! test -d "$NEW_DIR") && continue
+        DIST_VERSION=$(basename "$NEW_DIR")
+        DIST_NAME=$(basename $(dirname "$NEW_DIR"))
+        for arch in "amd64" "arm64"; do
+            mkdir -p "./tmp/${DIST_NAME}_${DIST_VERSION}_${arch}-unsigned"
+            tar -zxvf "./releases/${DIST_NAME}/${DIST_VERSION}/${DIST_NAME}_${DIST_VERSION}_darwin-${arch}.tar.gz" -C "./tmp/${DIST_NAME}_${DIST_VERSION}_${arch}-unsigned/"
+        done
+    done
+    ls -Rhl ./tmp || echo "Nothing new in ./tmp"
+echo "::endgroup::"
+
+echo "::group::Sign and notarize the mac binaries"
+    # Find and sign executables in
+    # ./tmp/${DIST_NAME}_${DIST_VERSION}_${arch}-unsigned/
+    for NEW_DIR in ./releases/*/*; do
+        (! test -d "$NEW_DIR") && continue
+        DIST_VERSION=$(basename "$NEW_DIR")
+        DIST_NAME=$(basename $(dirname "$NEW_DIR"))
+        for arch in "amd64" "arm64"; do
+            EXECUTABLES=$(jq -nc '$ARGS.positional' --args $(find "./tmp/${DIST_NAME}_${DIST_VERSION}_${arch}-unsigned/" -perm +111 -type f -print))
+            echo "{
+                \"source\" : $EXECUTABLES,
+                \"bundle_id\" : \"io.ipfs.dist.${DIST_NAME}\",
```

Comment:
:information_source: I registered the wildcard `bundle_id=io.ipfs.dist.*`  at Apple, which enables us to have each package "isolated" from each other. This is important in case Apple decides to block something from running on macOS – we don't have all egs in one basket.